pysolarfocus/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Python client lib for Solarfocus"""
-__version__ = "0.99.0"
+__version__ = "3.1.0"
```

Comment:
Maybe you should think about automating the versioning of this package.
You could use a github action to extract the version from the release tag while building the pip release.

Reply:
Like it #18